CONTRIBUTING.md

```diff
@@ -251,6 +251,20 @@ To keep naming patterns consistent across the project, naming patterns are enfor
   - `func CreateTracesExport(...) {...}`
   - `func CreateTracesToTracesFunc(...) {...}`
 
+### Enumerations
+
+To keep naming patterns consistent across the project, enumeration patterns are enforced to make intent clear:
+
+- Enumerations should be defined using a type alias
```

Comment:
I don't agree with this. I think it should be type definition as we have already implemented across the project

Reply:
You're right, I've used the wrong term